src/app/view/modules/object/object.component.ts

```diff
@@ -97,4 +104,39 @@ export class ObjectComponent implements OnChanges, OnInit {
     }
```

Comment:
Probably this is a known, still not solved issue with `ngOnChanges` and `ngOnInit`, see here: https://github.com/angular/angular/issues/7782

They propose to use only one or the other. Another workaround that is listed at the end suggests to ignore the first call to `ngOnChanges`:

```
ngOnChanges(changes:{[key: string]: SimpleChange}) {
    if (!changes["myProp"].isFirstChange()) {
      // insert onchanges code
    }
  }
```